app/views/admin/_admin_options.html.erb

```diff
@@ -10,15 +10,16 @@
   <% end %>
 
   <h3 class="landmark heading"><%= ts("Admin Actions") %></h3>
-  <ul class="management actions" role="menu">
+  <ul class="management actions">
     <% if item.hidden_by_admin? %>
       <li>
         <%= link_to ts("Make %{item} Visible", item: item_type_name),
                     { controller: "admin/user_creations",
                       action: "hide",
                       id: item,
                       creation_type: item.class,
-                      hidden: 'false' } %>
+                      hidden: "false" },
+                    method: :put %>
```

Comment:
Philosophically, I'd say this was a POST (i.e. generic request to change something) rather than a PUT (usually a specific request to add a new or updated item to a collection), but I assume there was a reason for this change. (This obviously applies to the addition to `routes.rb` too)

Reply:
Just noting for any curious parties that we decided this was a level of philosophical debate we didn't want to get into on a Tuesday morning, so we're sticking with PUT now.